templates/taxi/manufacturer_list.html

```diff
@@ -23,11 +24,20 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                  <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
+                    <i class="bi bi-pencil"></i>
+                  </a>
+                  <a href="{% url "taxi:manufacturer-delete" pk=manufacturer.id %}">
```

Comment:
Inconsistent use of quotes in the `{% url %}` tag. It's better to use single quotes consistently, as in `{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}`.